cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/ServicePlanEntity.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.client.v2.serviceplans;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * The entity response payload for Service Plans
+ */
+@Data
+public final class ServicePlanEntity {
+
+    /**
+     * The visible flag
+     *
+     * @param public the visible flag
+     * @return the visible flag
+     */
+    private final boolean active;
```

Comment:
@antechrestos We also stick to boxed types (`Boolean` vs. `boolean`) because the documentation about what's guaranteed to be non-null is so poor.